anubis-management-api/src/policies/operations.py

```diff
@@ -35,10 +38,13 @@ def get_policies_by_service_path(
         if resource_type:
             db_policies = db_policies.filter(
                 models.Policy.resource_type.contains(resource_type))
+        if token:
+            db_policies = filter_policies_by_token_data(db_policies, tenant, token)
```

Comment:
i would refer to "filter policies by user profile", the rational is that, beside the jwt token, if needed this could be achieved using other means, e.g. a certificate. in which case, the "router" transform this to whatever format is suitable format. secondly, we should not pass here the whole token that contains other information that are not useful, but only the relevant ones.